.vib/logstash/goss/vars.yaml

```diff
@@ -0,0 +1,27 @@
+binaries:
+  - java
+  - logstash
+  - logstash-plugin
+  - yq
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/logstash
+      - /bitnami/logstash/config
+      - /bitnami/logstash/data
+      - /bitnami/logstash/pipeline
+      - /opt/bitnami/logstash
+      - /opt/bitnami/logstash/config
+      - /opt/bitnami/logstash/pipeline
+      - /opt/bitnami/logstash/logs
+      - /opt/bitnami/logstash/plugins
+      - /opt/bitnami/logstash/tmp
+      - /docker-entrypoint-initdb.d
+files:
```

Comment:
I would add the following files here, with their permissions check:

- /opt/bitnami/logstash/Gemfile
- /opt/bitnami/logstash/Gemfile.lock

https://github.com/bitnami/containers/blob/34ae65e2f29bc7f97111aa8895af62d04b16c068/bitnami/logstash/7/debian-11/rootfs/opt/bitnami/scripts/logstash/postunpack.sh#L25-L27

Reply:
Added